contents/tutorials/web-worker.md

```diff
@@ -0,0 +1,221 @@
+---
+title: How to use PostHog in a web worker
+date: 2024-12-18
+author:
+ - ian-vanagas
+tags:
+  - configuration
+---
+
+Web workers enable you to run computationally expensive tasks or scripts in background threads. This prevents the main webpage thread from slowing down or being blocked. 
+
+Because the web worker works on a background thread, it can't access the DOM, browser APIs, or window object methods. Instead, it communicates with the main thread through a messaging system .
```

Comment:
```suggestion
Because the web worker works on a background thread, it can't access the DOM, browser APIs, or window object methods. Instead, it communicates with the main thread through a messaging system.
```